scripts/aggregate-verdict.py

```
@@ -195,6 +197,123 @@ def aggregate(verdicts: list[dict], override: Override | None = None) -> dict:
+        if vd in model_stats[model]["verdicts"]:
+            model_stats[model]["verdicts"][vd] += 1
+        if v.get("runtime_seconds") is not None:
+            model_stats[model]["total_runtime_seconds"] += v["runtime_seconds"]
+            model_stats[model]["runtimes"].append(v["runtime_seconds"])
+        if v.get("fallback_used"):
+            model_stats[model]["fallback_count"] += 1
+        if is_fallback_verdict(v):
+            model_stats[model]["parse_failures"] += 1
+
+    # Compute averages and rates per model
+    for model, stats in model_stats.items():
```

Ruff (0.15.0)

[warning] Loop control variable `model` not used within loop body

(B007)